docs/ai-coder/agent-boundary.md

```diff
@@ -4,118 +4,50 @@ Agent Boundaries are process-level firewalls that restrict and audit what autono
-
-Agents are prevented from reaching restricted domains or exfiltrating data, without blocking the rest of the dev's environment.  
-
-This is the fastest way to add real guardrails, but a determined user could still operate a tool outside of Boundary restrictions because the broader environment allows it. This mode relies on tools respecting certain settings, like HTTP proxies, and can lead to silent failures if a tool bypasses them.
-
-#### Privileged Mode
-
-In this case, boundaries are enforced at the level of the environment that the agent lives in. These are workspace- or session-level controls, including how the developer connects to it.  
-
-Currently, this must be turned on with a flag and ran with higher-level permissions such as root access or `CapNetAdmin`.
-
-In addition to process-level egress rules, privileged mode locks down all pathways that could bypass policy, such as restricting or disabling SSH tunnels or parallel unbound IDEs. This delivers deterministic, policy-as-code enforcement and offers the highest assurance for regulated environments, but results in slightly more friction for mixed human-and-agent workflows.
-
-### Opting out of Boundary
+The easiest way to use Agent Boundaries is through existing Coder modules, such as the [Claude Code module](https://registry.coder.com/modules/coder/claude-code). It can also be ran directly in the terminal by installing the [CLI](https://github.com/coder/boundary).
 
-If you tried Boundary through a Coder module and decided you don't want to use it, you can turn it off by setting the flag to `boundary_enabled=false`.
+Below is an example of how to configure Agent Boundaries for usage in your workspace.
+
+```
+module "claude-code" {
+  source              = "dev.registry.coder.com/coder/claude-code/coder"
+  enable_boundary     = true
+  boundary_version    = "main"
+  boundary_log_dir    = "/tmp/boundary_logs" 
+  boundary_log_level  = "WARN"
+  boundary_additional_allowed_urls = ["GET *google.com"]
+  boundary_proxy_port = "8087"
+  version             = "3.2.1"
+  ...
+}
+```
+
+- `boundary_version` defines what version of Boundary is being applied. This is set to `main`, which points to the main branch of `coder/boundary`.
+- `boundary_log_dir` is the location of the file where requests are logged when the workspace spins up.
+- `boundary_log_level` defines the verbosity at which requests are logged. Boundary uses the following verbosity levels:
+  - `WARN`: logs only requests that have been blocked by Boundary
+  - `INFO`: logs all requests at a high level
+  - `DEBUG`: logs all requests in detail
+- `boundary_additional_allowed_urls`: defines the URLs that the agent can access, in additional to the default URLs required for the agent to work
+  - `github.com` means only the specific domain is allowed
+  - `*.github.com` means only the subdomains are allowed - the specific domain is excluded
+  - `*github.com` means both the specific domain and all subdomains are allowed
```

Comment:
You can also specify specific HTTP methods separated by commas and specific paths. `GET,HEAD *github.com/coder`, for example. You don't have to go into detail, just a mention like "you can filter on methods, hostnames, and paths"